recipe/meta.yaml

```diff
@@ -19,14 +19,15 @@ requirements:
     - cmake
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
     - ninja  # [win]
     - make  # [not win]
   host:
     - python
     - numpy
     - xtensor >=0.24,<0.25
     - xtensor-python >=0.26,<0.27
-    - vigra 1.11.1
+    - libvigra
```

Comment:
this is the C dependencies, the python stuff is `vigra`

Reply:
You might need `vigra` if you import it in your setup script. But maybe add both??? It seems that maybe you link to vigranumpy.

Reply:
If the tests run through this change should be fine.